### ISSUE TITLE
Missing pre-processing for Mobilenet V2 model training

### ISSUE THREAD

**Issue by getsanjeevdubey** [tensorflow/tensorflow#39925]
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
https://colab.research.google.com/github/tensorflow/examples/blob/master/community/en/flowers_tf_lite.ipynb

https://github.com/tensorflow/examples/blob/master/lite/codelabs/flower_classification/android/finish/app/src/main/java/org/tensorflow/lite/examples/classification/tflite/Classifier.java#L292

## Description of issue (what needs changing):
In the above codelabs tutorial, we see image has been rescaled to [0-1] by dividing by 255. Since pre-trained weights (imagenet) are trained by feed [-1 1] normalized image. Ideally tutorial should add that step to correctly leverage transfer learning. 

### Clear description
So what is happening is we create a tflite model trained with [0-1] based preprocessing. On android client we are doing [-1 1] based preprocessing before feeding to tflite model.
Can someone please clarify?
### Correct links

Is the link to the source code correct?

### Parameters defined

Are all parameters defined and formatted correctly?

### Returns defined

Are return values defined?

### Raises listed and defined

Are the errors defined? For example,
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/feature_column/categorical_column_with_vocabulary_file#raises

### Usage example

Is there a usage example?

See the API guide: https://www.tensorflow.org/community/contribute/docs_ref
on how to write testable usage examples.

### Request visuals, if applicable

Are there currently visuals? If not, will it clarify the content?

### Submit a pull request?

Are you planning to also submit a pull request to fix the issue? See the docs
contributor guide: https://www.tensorflow.org/community/contribute/docs,
docs API guide: https://www.tensorflow.org/community/contribute/docs_ref and the
docs style guide: https://www.tensorflow.org/community/contribute/docs_style

**Comment by getsanjeevdubey** [comment]
@yashk2810 Could you please help with this?

**Comment by khanhlvg** [comment]
@xunkai55 Can you clarify why the TF Lite image classification seems to normalize image input to [-1, 1] instead of [0, 1]?
https://github.com/tensorflow/examples/blob/master/lite/codelabs/flower_classification/android/finish/app/src/main/java/org/tensorflow/lite/examples/classification/tflite/Classifier.java#L292

**Comment by getsanjeevdubey** [comment]
@khanhlvg Thanks for the response. I will look into TF Lite Model Maker.
Just to make my point clear, the issue is with: not having **same** preprocessing for both pipelines.
Had the original model was trained on [-1, 1] it would have been correct.

**Comment by khanhlvg** [comment]
You're right. The image classification model in the codelab is trained with Model Maker, which emits models required input normalization to [0, 1]. We have fixed the Android app accordingly. Thanks for reporting the issue.

https://github.com/tensorflow/examples/commit/bb26df46b498eb1a67630f29619705eaa043528f
